src/server/ua_subscription_events_filter.c

```diff
@@ -416,7 +460,9 @@ compareOperation(UA_Variant *firstOperand, UA_Variant *secondOperand, UA_FilterO
         UA_TYPES_DIFFERENT_NUMERIC_FLOATING_POINT,
         UA_TYPES_DIFFERENT_TEXT,
         UA_TYPES_DIFFERENT_COMPARE_FORBIDDEN,
-        UA_TYPES_DIFFERENT_COMPARE_EXPLIC
+        UA_TYPES_DIFFERENT_COMPARE_EXPLIC,
```

Comment:
The reasoning of this change is pretty difficult to communicate to future developers.
At least add a couple of comments to the cryptic enum definitions...